sensu/plugins/check-inspec.rb

```diff
@@ -60,18 +60,21 @@ def run
     results = inspec(config[:controls], config[:attrs])
     passed = 0
     failed = 0
+    skipped = 0
     msg = ""
     results['controls'].each do |control|
       #puts control
       if control['status'] == "passed"
         passed += 1
+      elsif control['status'] == "skipped"
```

Comment:
Why not just look for a "failed" status. That way you do not need to worry about "skipped" status. looks like you are not using the skipped variable anywhere else.